app/Http/Controllers/WikiController.php

```diff
@@ -6,7 +6,7 @@
 use App\Jobs\MediawikiInit;
 use App\Jobs\ProvisionQueryserviceNamespaceJob;
 use App\Jobs\ProvisionWikiDbJob;
-use App\Jobs\ElasticSearchIndexInit;
+use App\Jobs\ElasticSearch\ElasticSearchIndexInit;
```

Comment:
This should probably say CirrusSearch instead of ElasticSearch as this is what it's interacting with.

Reply:
Sounds good to me!